### ISSUE TITLE
Typescript: Incorrect types for accessibility 'enabled' properties (v8.0.0)

### ISSUE THREAD

**Issue by leettaylor** [highcharts/highcharts#12642]
There's a few properties for some of the Accessibility options which appear to be incorrectly labelled as `object` instead of `boolean`. If there are intended to be `object` the documentation for these options should be updated to explain the correct usage.

https://api.highcharts.com/highcharts/exporting.accessibility.enabled
https://api.highcharts.com/highcharts/legend.accessibility.enabled
https://api.highcharts.com/highcharts/legend.accessibility.keyboardNavigation.enabled

### `highcharts.d.ts`
```javascript
export interface ExportingAccessibilityOptions {
    /**
     * (Highcharts, Highstock, Highmaps, Gantt) Enable accessibility support for
     * the export menu.
     */
    enabled?: object;
}

// ....

export interface LegendAccessibilityKeyboardNavigationOptions {
    /**
     * (Highcharts, Highstock, Highmaps, Gantt) Enable keyboard navigation for
     * the legend.
     */
    enabled?: object;
}

// ....

export interface LegendAccessibilityOptions {
    /**
     * (Highcharts, Highstock, Highmaps, Gantt) Enable accessibility support for
     * the legend.
     */
    enabled?: object;

    // ....
}
```

**Comment by sebastianbochan** [comment]
Hi @cexbrayat,
Thank you for reporting about the problem. 

FYI @bre1470
